packages/shortcuts-extension/style/base.css

```diff
@@ -0,0 +1,418 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+/* Shortcut Input Style */
+
+.jp-Shortcuts-InputBox {
+  display: inline-flex;
+  padding-top: 2px;
+}
+
+.jp-Shortcuts-InputBoxNew {
+  margin-left: 10px;
+}
+
+.jp-mod-hidden {
+  display: hidden;
+}
+
+@keyframes slide-animation {
+  from {
+    width: 0;
+    left: 0;
+  }
+
+  to {
+    width: 120px;
+    left: 0;
+  }
+}
+
+.jp-Shortcuts-Input {
+  animation-duration: 0.5 s;
+  animation-timing-function: ease-out;
+  animation-name: 'slide-animation';
+  border-width: var(--jp-border-width);
+  border-color: var(--jp-border-color3);
+  border-style: solid;
+  background-color: var(--jp-layout-color0);
+  margin-left: auto;
+  padding-left: 10px;
+  width: 120px;
+  height: 25px;
+  line-height: 25px;
+  display: block;
+}
+
+.jp-Shortcuts-Input:focus {
+  outline: none;
+  color: var(--jp-content-font-color1);
+  border-color: var(--jp-brand-color2);
+}
+
+.jp-mod-unavailable-Input:focus {
+  border-color: var(--jp-error-color2);
+}
+
+.jp-Shortcuts-InputText {
+  overflow-x: hidden;
+  overflow-y: hidden;
+  margin: 0;
+  margin-top: 4px;
+  padding: 0 5px;
+  height: 17px;
+  line-height: 17px;
+  width: fit-content;
+}
+
+.jp-mod-selected-InputText {
+  background-color: var(--jp-brand-color3);
+  overflow: hidden;
+}
+
+.jp-mod-waiting-InputText {
+  color: var(--jp-content-font-color3);
+}
+
+.jp-Shortcuts-Submit {
+  background: var(--jp-brand-color2);
+  border-radius: 0;
+  border: none;
+  color: var(--jp-layout-color0);
+  font-family: var(--jp-ui-font-family);
+  display: block;
+  height: 27px;
+  background-image: var(--jp-icon-checkmark-white);
+  background-repeat: no-repeat;
+  width: 26px;
+}
+
+.jp-Shortcuts-Submit:focus {
+  outline: none;
+}
+
+.jp-mod-defunc-Submit {
+  background: var(--jp-layout-color3);
+  background-image: var(--jp-icon-checkmark-white);
+}
+
+.jp-mod-conflict-Submit {
+  background: var(--jp-error-color1);
+  background-image: var(--jp-icon-error-white);
```

Comment:
There is no such an icon, as there is no `jp-icon-checkmark-white`. It seems that it was not brought in when the extension got merged into the core, so it is not a problem in this PR. Would you mind (in order of preference):
- using a different icon (e.g. `jp-icon-check` or `jp-icon-save` for `jp-icon-checkmark-white`)/adding an icon if needed, or
- removing the icon and just adding text in the appropriate buttons, or
- adding a TODO comment when non-existing icons are referenced?

Reply:
Currently the buttons looks like this:

![Screenshot from 2022-12-17 16-59-53](https://user-images.githubusercontent.com/5832902/208252968-010cd056-57bb-41a2-bb15-4ddf6ab7f432.png)

![Screenshot from 2022-12-17 17-00-36](https://user-images.githubusercontent.com/5832902/208252995-9e81fb94-4349-4965-a100-57453c7d848f.png)

Also it appears we don't have an UI screenshots for setting editor; I would not want to ask for too much here - if it is out of scope, we can create a "good first issue" to add them.

Reply:
Thanks for the review @krassowski 

The latest commit add/restore proper icons for the shortcut input elements and the search input (switch to use `InputGroup` for consistency).
I also switch to use icon for the sort indicator on the table header.

I will leave the UI screenshots out of this one.